packages/react-reconciler/src/ReactFiberHydrationContext.new.js

```diff
@@ -601,10 +607,28 @@ function resetHydrationState(): void {
   isHydrating = false;
 }
 
+export function queueRecoverableHydrationErrors(): void {
```

Comment:
These two methods sound the same. Maybe "apply" or "upstream"...?

Reply:
upgradeHydrationErrorsToRecoverable?